src/object/srv_obj_migrate.c

```diff
@@ -2481,7 +2515,7 @@ migrate_one_epoch_object(daos_epoch_range_t *epr, struct migrate_pool_tls *tls,
 	if (daos_oclass_is_ec(&unpack_arg.oc_attr)) {
 		p_csum = NULL;
 		/* EC rotate needs to fetch from all shards */
-		if (unpack_arg.ec_rotate_parity)
+		if (arg->oid.id_layout_ver > 0)
```

Comment:
maybe a helper of "obj_ec_shard_rotate_anabled" is easier to follow.

Reply:
sure